collin/styles.css

```
@@ -0,0 +1,72 @@
+body {
+    text-align: center;
+    margin-top: 3em;
+    font-family: "Open Sans", sans-serif;
+  }
+  
+  #title {
+    font-size: 3em;
+  }
+  
+  .joke-setup {
+    margin: 3em;
+  }
+  
+  .joke-punchline {
+    display: none;
+    margin: 3em;
+  }
+  
+  .joke-punchline img {
+    width: 10em;
+    display: block;
+    margin-top: 2em;
+    margin-left: auto;
+    margin-right: auto;
+    -webkit-animation: spin 5s linear infinite;
+    -moz-animation: spin 5s linear infinite;
+    animation: spin 5s linear infinite;
+  }
+  
+  @-moz-keyframes spin {
+    100% {
+      -moz-transform: rotate(360deg);
+    }
+  }
+  @-webkit-keyframes spin {
+    100% {
+      -webkit-transform: rotate(360deg);
+    }
+  }
+  @keyframes spin {
+    100% {
+      -webkit-transform: rotate(360deg);
+      transform: rotate(360deg);
+    }
+  }
+  
+  .punchline-box:hover + .joke-punchline {
```

In order to accomodate keyboard users, you could add to this CSS rule so that `:focus` also works!

```suggestion
  .punchline-box:hover + .joke-punchline, .punchline-box:focus + .joke-punchline {
```

---

You'll need to add something called a `tabindex` to your HTML tag you want to `:focus` on to make this work. Give it a Goog

---

will def goog